js/src/analysis/keywordForms.js

```diff
@@ -0,0 +1,19 @@
+import { Paper } from "yoastseo";
+import { updateKeywordForms } from "../redux/actions/snippetEditor";
+
+/**
+ * Gets morphological forms of the focus keyword and makes them ready for yoast-components.
```

Comment:
`Gets` is more for when you just get something that is stored. What happens here is: The morphologic research is run and the results are saved to the store. I would suggest using `Calculates`.
Instead of `ready for yoast-components` I would also suggest a more precise `ready for the snippet editor`.